scalafmt-tests/src/test/resources/align/DefaultWithAlign.stat

```diff
@@ -420,3 +420,13 @@ Seq(
   "org.scalatest"      % "scalatest_2.11"               % "3.0.0" % Test,
   "org.scalamock"      %% "scalamock-scalatest-support" % "3.2.2" % Test
 )
+<<< align coments in method chain
```

Comment:
Can we add more tests to see if the change has unintended consequences? For example, I'm curious to know what happens in the following cases

```scala
// Test 1: separate statements
val ys = xs
  .filter(_ > 2) // comment1
  .map(x => x + 1) // comment2
ys.map(y => y + 1) // comment3

// Test 2: Missing handling for Term.ApplyType
foo
   .x[T]() // comment1
   .xx[T]() // comment2

// Test 3: iirc, this didn't align before but maybe it's been fixed now
a.b(
  1, // comment1
  1 // comment2
)

// Test 4: alignment of random dots

a.b.c.d
aa.bb.cc.dd

// Test 4: alignment of random parentheses

a().b().c().d
aa().bb().cc().dd
```

Have you tried running this change on your codebase? I would be interested to see a large diff from a representative codebase. I personally don't use vertical alignment so I don't know what behaviour makes most sense.

Reply:
OK, I'll try running this chage on some codebases 👍  (I've done only on my private project yet...) and add some tests like you mentioned! Thank you for your advice.